judgels-client/src/modules/api/gabriel/language.js

```diff
@@ -14,6 +14,7 @@ export const gradingLanguageNamesMap = {
   Python3: 'Python 3',
   PyPy3: 'PyPy 3',
   Rust2021: 'Rust 2021',
+  JavascriptNode18: 'Javascript (Node.js 18)',
```

Comment:
Nit: the languages here (and other occurrences in this file) are sorted alphabetically, can you also do that for the new JavaScript language?